### PR TITLE
Performance Improvement in the Handbook section

### LINKED ISSUE #3014: [Performance] Image Optimization <img> to <StaticImages> in the handbook section
**Current Behavior**

The .png and .jpg images cause a lot of performance issues 
**Desired Situation**

Change `` to `<StaticImagess>` for image optimization in the handbook section.  eg [PR](https://github.com/layer5io/layer5/pull/2597)  [StaticImagePlugin](https://www.gatsbyjs.com/docs/how-to/images-and-media/using-gatsby-plugin-image/)
---
[Optional] **Alternatives**

[Optional] **Additional context**

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by legendarykamal**
**Description**

This PR fixes #3014

**Notes for Reviewers**
Whole handbook img set to StaticImages inorder to improve performance.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Community/Calendar/index.js [hunk 1]
```diff
@@ -129,6 +132,28 @@ const Calendar = () => {
         </Container>
         <Table columns={columns} data={data} />
       </div>
+      <br/>
+      <Container>
```
Comment (Nikhil-Ladha) [inline src/sections/Community/Calendar/index.js:136]: Why this section is added?

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
Add `alt` to each img that you have updated, which doesn't have one already.

Also, you have not contributed enough to know what a meaningful PR title should be! Please update it and keep that in mind from your next PR onwards.

File: src/sections/Community/Calendar/index.js (hunk 1 above)
Comment (legendarykamal) [reply src/sections/Community/Calendar/index.js]: I have deleted that file, It wasn't a part of this PR.

**Comment by legendarykamal** [comment]
But @Nikhil-Ladha  wasn't that intended from this issue https://github.com/layer5io/layer5/issues/3014 
The whole handbook to be considered? I know there are many changes in files but that covers the whole handbook right ? 
or you intended something else @YashKamboj ?

**Comment by Nikhil-Ladha** [comment]
> But @Nikhil-Ladha wasn't that intended from this issue #3014 The whole handbook to be considered? I know there are many changes in files but that covers the whole handbook right ? or you intended something else @YashKamboj ?

I said "PR title" not just "PR".
Also, the file shouldn't be deleted only the changes that you have added should be.

**Comment by legendarykamal** [comment]
ohh  yeah got it title title (i missed) thanks for the review.

**Comment by legendarykamal** [comment]
ok finally I think I am done with this task , Please review

File: src/sections/Community/Handbook/community.js [hunk 2]
```diff
@@ -172,35 +175,35 @@ const CommunityGuide = () => {
             </p>
             <ul style={badgeListStyle}>
               <li>
-                <img src={CommunityLogo} style={badgeStyle} />
+                <StaticImage src={CommunityLogo} style={badgeStyle} alt="badgeStyle"/>
```
Comment (Nikhil-Ladha) [inline src/sections/Community/Handbook/community.js:178]: `alt` text over here should be `Community Logo`.
Please update similarly in other places as well

File: src/sections/Community/Handbook/connect.js [hunk 3]
```diff
@@ -200,8 +201,8 @@ const Connect = () => {
             </p>
             <div className="channels-list">
               <p className="channels-para">
-                <img
-                  className="channels-img"
+                <StaticImage
+                  className="channels-StaticImage"
```
Comment (Nikhil-Ladha) [inline src/sections/Community/Handbook/connect.js:205]: Update the same in the handbook.style.js file too

**Comment by Nikhil-Ladha** [comment]
Also, there are a couple of merge conflicts. Please resolve them too.

File: src/sections/Community/Handbook/connect.js (hunk 3 above)
Comment (legendarykamal) [reply src/sections/Community/Handbook/connect.js:205]: done :+1:

File: src/sections/Community/Handbook/community.js (hunk 2 above)
Comment (legendarykamal) [reply src/sections/Community/Handbook/community.js]: :+1:

**Comment by legendarykamal** [comment]
So many changes so many addittion and now everything is resolved :100: 
@Nikhil-Ladha @leecalcote are we good to merge now ?

**Review by Nikhil-Ladha — APPROVED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit ec3b88ab0976fbc713d815405a129b1902e6756a at: https://62f5ece340589074becd6bba--layer5.netlify.app

**Comment by debo19** [comment]
We don't have to use `<StaticImage>` for svgs right? This PR has disturbed the styles in various places
Screenshot 2022-08-12 at 5 46 06 PM
Screenshot 2022-08-12 at 5 48 05 PM

**Comment by debo19** [comment]
@Nikhil-Ladha should we revert this PR for now?

**Comment by legendarykamal** [comment]
oh sorry yes @debo19 we dont need changes for svg's Let me fix this in another PR right away.

**Comment by legendarykamal** [comment]
@debo19 I think some minor style changes are only required more.
As far as svg's are concerned then the img tag is mostly used for them only in most places.

**Comment by Nikhil-Ladha** [comment]
> We don't have to use `<StaticImage>` for svgs right? This PR has disturbed the styles in various places Screenshot 2022-08-12 at 5 46 06 PM Screenshot 2022-08-12 at 5 48 05 PM

Huh!
My bad 🥲

**Comment by legendarykamal** [comment]
ok @Nikhil-Ladha @debo19 so now should I make a fresh PR for this ? with excluding the svg's
Without svg's this was much easier :sweat:

**Comment by debo19** [comment]
@legendarykamal I'll revert this PR so that you don't have to go through each svg again. After that, please open a new PR adding missing `alt` to any image and use `StaticImage` only for pngs/JPEGs, if any.

**Comment by legendarykamal** [comment]
No problem @debo19 I have finished doing it too in the meantime, let me request u for a review :smiley:

**Pull request merged**